### ISSUE TITLE
How to expose DocumentDirectoryPath to my app?

### ISSUE THREAD

**Issue by souljacker** [itinance/react-native-fs#772]
I'm using react-native-fs to store a file in my cache directory by using the following code:

  ```
downloadFile = (url, name) => {
    const destPath = RNFS.DocumentDirectoryPath + '/' + name;
    let option = {
      fromUrl: url,
      toFile: destPath,
    };
    RNFS.downloadFile(option).promise.then(res => {
      console.log('res -----------------------------> ', res);
      RNFS.readDir(RNFS.DocumentDirectoryPath) // On Android, use "RNFS.DocumentDirectoryPath" (MainBundlePath is not defined)
        .then(result => {
          console.log('GOT RESULT', result);
        });
    });
  };
```

The files are being stored properly, the full path being /data/user/0/com.scpush/files/image.jpg

I'm now passing this full path to a function I'm exposing to RN via Native Modules and that file can't be opened due to this error

<img width="368" alt="Screen Shot 2019-09-27 at 00 13 09" src="https://user-images.githubusercontent.com/8139090/65840144-99616680-e2eb-11e9-82cb-94bec727c099.png">

I figure I must expose that directory to my app, and I googled around, and the best I could come up with was these two files:

**android/app/src/main/res/xml/files_paths.xml**

```
<?xml version="1.0" encoding="utf-8"?>
<paths>
  <external-path
    name="external"
    path="." />
  <external-files-path
    name="external_files"
    path="." />
  <cache-path
    name="cache"
    path="." />
  <external-cache-path
    name="external_cache"
    path="." />
<files-path
    name="files"
    path="." />
</paths>
```

**android/app/src/main/res/xml/provider_paths.xml**

```
<?xml version="1.0" encoding="utf-8"?>
<paths xmlns:android="http://schemas.android.com/apk/res/android">

</paths>
```

I'm sure I'm totally messing up here, but I can't figure where. I'm very new to RN.

In other words, how can I expose DocumentDirectoryPath to my app?

Can someone give me some help?
